dbld/images/kira.dockerfile

```diff
@@ -1,7 +1,10 @@
 FROM balabit/syslog-ng-ubuntu-focal
 LABEL maintainer="Andras Mitzki <andras.mitzki@balabit.com>, Laszlo Szemere <laszlo.szemere@balabit.com>, Balazs Scheidler <balazs.scheidler@oneidentity.com>"
 
-ENV OS_PLATFORM kira
+ARG IMAGE_PLATFORM
+ARG COMMIT
+ENV IMAGE_PLATFORM kira
```

Comment:
Why don't we use `${IMAGE_PLATFORM}` ? Here, and other places, too.
Or alternatively we don't need to use `ARG`s anywhere, and burn in the values to every dockerfile like we do it for kira.

Reply:
I've checked and there is no reason to manually override `IMAGE_PLATFORM` in case of `kira` or `tarball` or `devshell`. We can use the environment variable like in other platforms:

```
dbld/images/ubuntu-bionic.dockerfile:ENV IMAGE_PLATFORM ${IMAGE_PLATFORM}
```

Reply:
We discussed IRL, with the same name can't be set (because of inheritance), but If ARG variable name is different of ENV variable can be set (current version)